src/singletons/Settings.hpp

```diff
@@ -375,6 +375,14 @@ class Settings : public ABSettings, public ConcurrentSettings
         "/misc/twitch/messageHistoryLimit",
         800,
     };
+    IntSetting twitchMessageLogLimit = {
+        "/misc/twitch/messageLogLimit",
+        8000,
+    };
+    IntSetting twitchMessageOnScreenLimit = {
+        "/misc/twitch/messageOnScreenLimit",
+        1000,
+    };
```

Comment:
The setting controls the amount of scrollback in Splits
The setting does not do anything Twitch-specific
Setting name should be changed to reflect this
Variable name should be changed to reflect this